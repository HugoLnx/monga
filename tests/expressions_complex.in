int main() {
	x = new int[10];
	x = new int[(10)];
}
