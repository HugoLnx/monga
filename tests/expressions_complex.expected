program:
  function:main,259,0
    block:
      statements:
        statement:
          attribution:
            var:x
            exp:6
              new:259,0
                exp:0,10
        statement:
          attribution:
            var:x
            exp:6
              new:259,0
                exp:0,10
        statement:
          attribution:
            var:x
            exp:7
              binary:5
                exp:7
                  binary:5
                    exp:0,10
                    exp:0,15
                exp:0,20
        statement:
          attribution:
            var:x
            exp:7
              binary:6
                exp:7
                  binary:6
                    exp:0,10
                    exp:0,15
                exp:0,20
        statement:
          attribution:
            var:x
            exp:7
              binary:7
                exp:7
                  binary:7
                    exp:0,10
                    exp:0,15
                exp:0,20
        statement:
          attribution:
            var:x
            exp:7
              binary:8
                exp:7
                  binary:8
                    exp:0,10
                    exp:0,15
                exp:0,20
        statement:
          attribution:
            var:x
            exp:7
              binary:10
                exp:7
                  binary:10
                    exp:0,10
                    exp:0,15
                exp:0,20
        statement:
          attribution:
            var:x
            exp:7
              binary:9
                exp:7
                  binary:9
                    exp:0,10
                    exp:0,15
                exp:0,20
        statement:
          attribution:
            var:x
            exp:7
              binary:12
                exp:7
                  binary:5
                    exp:0,10
                    exp:0,15
                exp:7
                  binary:5
                    exp:0,5
                    exp:0,20
        statement:
          attribution:
            var:x
            exp:7
              binary:11
                exp:7
                  binary:5
                    exp:0,10
                    exp:0,15
                exp:7
                  binary:5
                    exp:0,5
                    exp:0,20
